ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/forkchoice/ForkChoice.java

```diff
@@ -582,6 +578,23 @@ private BlockImportResult importBlockAndState(
     return result;
   }
 
+  private boolean applyProposerBoost(
```

Comment:
given this returns boolean, i'd prefer to rename it... something like `isApplyProposerBoost`

`apply` suggests its changing something, but what this function doing is determine whether we *should* apply proposer boost.

I'd potentially go as far as referring to the spec formula we're computing by putting a comment above it like
```
// from consensus spec (maybe deneb/fork_choice?):
// if get_current_slot(store) == block.slot and is_before_attesting_interval and is_first_block:
//        store.proposer_boost_root = hash_tree_root(block)
private boolean isApplyProposerBoost(final SignedBeaconBlock block, final StoreTransaction transaction) {
```